src/Homebrew.jl

```diff
@@ -16,7 +16,11 @@ const brew_prefix = abspath(joinpath(dirname(@__FILE__),"..","deps", "usr"))
 const brew_exe = joinpath(brew_prefix,"bin","brew")
 const tappath = joinpath(brew_prefix,"Library","Taps","staticfloat","homebrew-juliadeps")
 
-const BREW_URL = "https://github.com/Homebrew/homebrew.git"
+if VERSION < v"0.5.0-dev+522"
+    const BREW_URL = "https://github.com/Homebrew/homebrew.git"
+else
+    const BREW_URL = "https://github.com/Homebrew/brew.git"
+end
```

Comment:
By "correctly" you mean updating to the newer version of `brew`?  Yes, that's true.  I'm currently setting up a clean 10.9 box so I can try this out.  I will put this "clean" VM on `macmini2` so we can test this kind of stuff out over VNC once I'm done.